arch.json

```diff
@@ -1,35 +1,63 @@
 {
-    "variables": {},
+    "_comment": "Keys prefixed with an underscore are comments.",
```

Comment:
As I don't have any comments I could remove this, but maybe it'll help someone down the line.